crates/uv/src/commands/tool/run.rs

```diff
@@ -147,12 +151,46 @@ pub(crate) async fn run(
                     "hint".bold().cyan(),
                     ":".bold(),
                     package_name.cyan(),
-                    format!("{} --from {} {}", invocation_source, package_name, target),
+                    format!("{invocation_source} --from {package_name} {target}").green(),
                 ))
             };
         }
     }
 
+    // If the user tries to invoke `uvx run ruff`, hint them towards `uvx ruff`, but only if
+    // the package is guaranteed to come from PyPI.
```

Comment:
```suggestion
    // the `run` package is guaranteed to come from PyPI.
```

Clarifies the motivation a little, could refer to `ruff` otherwise

Reply:
Might want to note `run` is reserved on PyPI?